libvmi/libvmi.h

```diff
@@ -2465,6 +2477,29 @@ status_t vmi_set_vcpuregs(
     registers_t *regs,
     unsigned long vcpu) NOEXCEPT;
 
+/**
+ * Allocates a new physical page.
+ *
+ * @param[in] vmi LibVMI Instance
+ * @param[in] gfn gfn of physical page to be allocated
+ */
+status_t vmi_alloc_gfn(
+    vmi_instance_t vmi,
+    uint64_t gfn) NOEXCEPT;
+
+
+/**
+ * Frees a with vmi_alloc_gfn allocated physical page.
```

Comment:
Typo, `with` is unnecessary. Technically this could free any gfn, not just the one allocated with vmi_alloc_gfn, right?

Reply:
On kvm, pages allocated via `vmi_alloc_gfn` are tracked, actually. Calling `vmi_free_gfn` on a page that has not been allocated via `vmi_alloc_gfn` will fail.